src/component/UploadFile/UploadFile.tsx

```diff
@@ -36,8 +38,19 @@ function UploadFile() {
   };
 
   return (
-    <div>
-      <input type="file" name="file" accept=".csv" onChange={handleChange} />
+    <div className="input">
+      <input
+        className="input--file"
```

Comment:
according to BEM conventions:

if input is the Block. then element inside input would be input__file and input__label

Reply:
Fixed the comment